.github/workflows/build_slim_packages.yaml

```diff
@@ -194,6 +194,11 @@ jobs:
         HTTP_PORT=$(docker inspect --format='{{(index (index .NetworkSettings.Ports "18083/tcp") 0).HostPort}}' $CID)
         ./scripts/test/emqx-smoke-test.sh localhost $HTTP_PORT
         docker stop $CID
+    - name: test two nodes cluster with proto_dist=inet_tls in docker
+      run: |
+        ./scripts/test/start-two-nodes-in-docker.sh -P $EMQX_IMAGE_TAG
+        # cleanup
+        ./scripts/test/start-two-nodes-in-docker.sh -c
```

Comment:
run smoke test against the haproxy endpoint ?

Reply:
Done.